cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-media.php

```diff
@@ -633,7 +633,7 @@ public function attachment_url( $url, $attachment_id ) {
 		if ( false !== $previous_url ) {
 			return substr( $url, $previous_url );
 		}
-		if ( ! doing_action( 'wp_insert_post_data' ) && false === $this->in_downsize ) {
+		if ( ! doing_action( 'wp_insert_post_data' ) && false === $this->in_downsize && ! apply_filters( 'cloudinary_doing_upload', '__return_false' ) ) {
```

Comment:
This filter needs a docblock!

Reply:
Also this filter will return the string `__return_false` (which would result in `true`). What you're looking for here is:

```suggestion
		if ( ! doing_action( 'wp_insert_post_data' ) && false === $this->in_downsize && apply_filters( 'cloudinary_doing_upload', false ) ) {
```

Reply:
@dugajean 

Great catch with the default value. 👍